my_dagster_project/assets/dbt_core/dbt_core.py

```diff
@@ -0,0 +1,14 @@
+from dagster import job, ScheduleDefinition, file_relative_path, define_asset_job, AssetSelection
+#from .fivetran_simple import fivetran_assets 
+#let's see if we need this as in theory the sources in the dbt project should get mapped
+from dagster_dbt import load_assets_from_dbt_project
+
+DBT_PROJECT_PATH = file_relative_path(__file__, "../../dbtproj")
```

Comment:
My bad - just pulled the image that is being built to confirm everything, and this should be `../../../dbtproj` 

(I missed the dbt_core folder earlier... )